### PR TITLE
Add DailyHabit UI

### PR DESCRIPTION
**Pull request by Docker-J**
## Description of Changes
Add Daily Habit UI with custom list

TODO: Background, Spinner(Dropdown day selector)

## Screenshots
Screenshot_1635994765

## Checklist

- [ ] Automated tests
- [x] Screenshots included (if necessary)
- [x] Code is well commented

Closes #ISSUE_ID

### CODE REVIEW

**Review by j-vent — DISMISSED** [review]
LGTM

File: src/app/src/main/java/com/cmput301f21t09/budgetprojectname/DailyHabitCustomList.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.ImageButton;
+import android.widget.TextView;
+
+import androidx.annotation.Nullable;
+
+import java.util.ArrayList;
+
+public class DailyHabitCustomList extends ArrayAdapter<HabitModel> {
```
Comment (MacMalainey) [inline src/app/src/main/java/com/cmput301f21t09/budgetprojectname/DailyHabitCustomList.java:16]: Please add comments

**Review by MacMalainey — CHANGES_REQUESTED** [review]
Please address the one issue otherwise LGTM

**Review by cheetiantey — APPROVED** [review]
LGTM

**Review by loravocado — APPROVED** [review]

**Comment by loravocado** [comment]
LGTM :D

**Review by MacMalainey — APPROVED** [review]

**Pull request merged**